Makefile

```diff
@@ -268,10 +272,10 @@ test-e2e-debug-cluster:
 	- oc logs deployment/cert-manager-operator -n cert-manager-operator
 	@echo "---- /Debugging the current state ----"
 .PHONY: test-e2e-debug-cluster
- 
+
 .PHONY: lint
-lint: 
-	$(GOLANGCI_LINT) run --config .golangci.yaml	
+lint:
```

Comment:
Not sure why these whitespace changes? (maybe your local linter is mis-behaving)

Reply:
Will undo these changes as they are not related to this PR. (Yeah. I've set the `"files.trimTrailingWhitespace": true` in my vscode editor)